public/app/features/plugins/admin/pages/PluginDetails.tsx

```diff
@@ -72,29 +72,24 @@ export default function PluginDetails({ match, queryParams }: Props): JSX.Elemen
     );
   }
 
-  return (
-    <Page>
+  const pageNav: NavModelItem = {
+    text: plugin.name,
+    img: plugin.info.logos.small,
+    breadcrumbs: [{ title: 'Plugins', url: parentUrl }],
+    children: tabs.map((tab) => ({
+      text: tab.label,
+      icon: toIconName(tab.icon ?? ''),
+      url: tab.href,
+      active: tab.id === pageId,
+    })),
```

Comment:
I think this is just quick example of what the model should look like, I think what we should do is change usePluginDetailsTabs so it returns NavModelItem (so this is built inside a useMemo) 
Would like help with this if possible

Reply:
@torkelo - took a stab at this and refactored the pluginDetails hook together with @mckn. I believe it now does what we need for this PR. Let me know if I've missed anything.

Reply:
@jackw that's great, thanks!